.github/workflows/test.yaml

```diff
@@ -34,6 +34,10 @@ jobs:
         run: |
           make test
 
+      - name: Sleep random seconds to ensure test call are not executed at exactly the same time
```

Comment:
Since this probably doesn't fix the serializable issues you could also reduce it to something like 20-30s or remove it entirely, but I don't really care.